client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -475,6 +476,8 @@ class DomainManagementNavigationEnhanced extends React.Component {
 			}
 		} else if ( domainTypes.MAPPED === domainType ) {
 			title = translate( 'Delete domain mapping' );
+		} else if ( domainTypes.SITE_REDIRECT === domainType ) {
+			title = translate( 'Delete site redirect' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Delete Site Redirect' )` *ES Score: 9*